test/built-ins/Proxy/get/trap-is-not-undefined-accessor-get-is-undefined-throws.js

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
```

Comment:
It would be more concise/consistent to omit `trap-is-not-undefined` from the name of this file. (We can take that for granted as the "normal" case.)